src/blocks.js

```diff
@@ -181,7 +181,14 @@
     "block_icon": "./blocks/vishalpatil20/icon.png",
     "author_name": "vishalpatil20",
     "author_github_url": "https://github.com/vishalpatil20"
-  }
+  },
```

Comment:
Please keep `src/blocks.js` as it is (see step 10 in the `README.md`).

Reply:
Done